src/core/Akka.Streams.Tests/Dsl/FlowSelectAsyncUnorderedSpec.cs

```diff
@@ -134,7 +134,7 @@ await this.AssertAllStagesStoppedAsync(async() => {
                     .To(Sink.FromSubscriber(c)).Run(Materializer);
                 var sub = await c.ExpectSubscriptionAsync();
                 sub.Request(10);
-                c.ExpectError().InnerException.Message.Should().Be("err1");
+                (await c.ExpectErrorAsync()).Message.Should().Be("err1");
```

Comment:
We no longer get `AggregateException`s here unless the test completes immediately, in which case we might have to harden this in the future.